oc_cdtapi/ForemanAPI/api.py

```
@@ -15,8 +15,10 @@
 
 class ForemanAPIError(HttpAPIError):
     def __str__(self):
-        return self.text
-
+        if self.resp is not None:
+            return 'Code: ' + str(self.code) + ' Message: ' + self.resp.text
+        else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)